tests/integration/upload_images.t

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+use ProductOpener::TestDefaults qw/:all/;
+
+use File::Basename "dirname";
+
+use Storable qw(dclone);
+
+remove_all_users();
+
+remove_all_products();
+
+wait_application_ready();
+
+my $ua = new_client();
+
+my %create_user_args = (%default_user_form, (email => 'bob@gmail.com'));
+create_user($ua, \%create_user_args);
+
+
+my $tests_ref = [
+    {
+        test_case => 'upload_image_nonexistent_product',
+        method => 'POST',
+        path => '/cgi/product_image_upload.pl',
+        form => {
+            code => 'nonexistent_product_code',
+            imgupload_front_en => ['/path/to/image.jpg', 'image.jpg']
```

Comment:
Here we need to reference the path of an existing image.
We can reuse an image from https://github.com/openfoodfacts/openfoodfacts-server/tree/main/tests/integration/inputs/sample-products-images/300/000/000/0001 like 1.jpg

To find the path: my $sample_products_images_path = dirname(__FILE__) . "/inputs/sample-products-images/";